modules/literal/src/main/scala-3/io/circe/literal/JsonLiteralMacros.scala

```diff
@@ -0,0 +1,98 @@
+package io.circe.literal
+
+
```

Comment:
Looks like we need to run scalafmt on these changes.  I see a few places missing spaces or having the space in the wrong place.  Can you correct that please?

Reply:
Sure!

I updated scalafmt to latest for this.